subsys/bluetooth/mesh/shell/shell_dm_cli.c

```diff
@@ -180,15 +180,18 @@ static int cmd_instance_set(const struct shell *shell, size_t argc, char *argv[]
 }
 
 SHELL_STATIC_SUBCMD_SET_CREATE(instance_cmds,
-			       SHELL_CMD_ARG(set, NULL, "<elem_idx> ", cmd_instance_set, 2, 0),
+			       SHELL_CMD_ARG(set, NULL, "<ElemIdx> ", cmd_instance_set, 2, 0),
 			       SHELL_CMD_ARG(get-all, NULL, NULL, cmd_instance_get_all, 1, 0),
 			       SHELL_SUBCMD_SET_END);
 
 SHELL_STATIC_SUBCMD_SET_CREATE(
-	dm_cmds, SHELL_CMD_ARG(cfg, NULL, "[<ttl> <timeout> <delay>]", cmd_cfg, 1, 3),
-	SHELL_CMD_ARG(start, NULL, "<mode> <addr> [<reuse-transaction> [<ttl> <timeout> <delay>]]",
-		      cmd_dm_start, 3, 4),
-	SHELL_CMD_ARG(result-get, NULL, "<entry_cnt>", cmd_result_get, 2, 0),
+	dm_cmds,
+	SHELL_CMD_ARG(cfg, NULL, "[<TTL> <Timeout(100ms steps)> <Delay(µs)>]", cmd_cfg, 1, 3),
+	SHELL_CMD_ARG(
+		start, NULL,
+		"<Mode> <Addr> [<ReuseTransaction> [<TTL> <Timeout(100ms steps)> <Delay(µs)>]]",
```

Comment:
Do we really do non-ASCII text in our C files? I'm actually asking, as I don't know if there is any agreed-upon practice here, but I'm used to that being avoided in other projects.

Reply:
Maybe @mia-ko can clarify this since it is her suggestion? Im not sure to be honest.

Reply:
Coding best practices take precedence in C files.
µ or micro has to be used in rst files (and probably Kconfig)

Reply:
I just thought of the fact that, in addition to being in C source code, this string will go through whatever shell backend is used. I think we really should assume the possibility of an ASCII-only backend, and change this back to `"us"`, at least now that Mia seems okay with it.